src/main/java/com/coveros/selenified/Selenified.java

```diff
@@ -549,7 +551,7 @@ void doSetup() throws InvalidBrowserException {
          *                                 Selenium.Browser class is used, this exception will be
          *                                 thrown
          */
-        private static void setupTestParameters() throws InvalidBrowserException {
+        private static void setupTestParameters() throws InvalidBrowserException, InvalidProxyException, InvalidHubException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.coveros.selenified.exceptions.InvalidHubException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)